README.md

```diff
@@ -294,17 +321,17 @@ In this example we search for all elements with the `fsType` equals `Example`. W
 
 ```typescript
 fsxaApi.fetchByFilter(
-    [
-      {
-        field: "fsType",
-        operator: ComparisonQueryOperatorEnum.EQUALS,
-        value: "Example"
-      }
-    ],
-    "en",
-    2,
-    50,
-    {"keys": {"identifier": 0}}
+  [
+    {
+      field: 'fsType',
+      operator: ComparisonQueryOperatorEnum.EQUALS,
+      value: 'Example'
+    }
+  ],
+  'en',
```

Comment:
Shouldnt it be 'en_EN' or 'de_DE'

Reply:
No, since the configured language for the default project is `en_GB`.